EasyPostTest/ClientFactoryTest.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using EasyPost;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace EasyPostTest
+{
+    [TestClass]
+    public class ClientFactoryTest {
+
+        private class FakeClientFactoryForKeyPlusUrl : IClientFactory {
```

Comment:
Getting a build error here in 4.5. "The type or namespace name 'IClientFactory' could not be found (are you missing a using directive or an assembly reference?)"